### DISCUSSION TITLE
Navigation menu reorganization

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2306] [Ideas]
I tried to rethink the menu to make it more intuitive and contextualized.
 
# Home 
# Getting Started
  - What's new
  - Upgrade guide
  - Code setup
  - Project templates

# Features
  - Themes
  - Design tokens
  - Reboot
  - Icons and Emoji
    - Emoji
    - Icon
  - Component ( actual Component / Overview)
  - Services
    - DialogService
    - MessageService
    - ToastService
    - KeyCodeService

# Layout
  - Header
  - Footer
  - BodyContent
  - Grid
  - Layout
  - MainLayout
  - Spacer
  - Splitter
    - Splitter
    - Splitter Multi
  - Stack
  - Card
  - Dialog
  - Divider
  - Panel
  - CollapsibleRegion (not in docuemntation)

# Form & Inputs
  - Overview
  - Autocomplete
  - Checkbox
  - InputFile
  - List
    - Combobox
    - Listbox
    - Select
    - Option
    - Sortable List
  - Number Field
  - Radio
  - Radio Group
  - Rating
  - Search
  - Slider
  - Switch
  - TextArea
  - Text Field
  - Date & Time
	  
# Navigation	  
  - Accordion
  - Anchor
  - Anchored Region
  - Breadcrumb
  - AppBar
  - Menu
  - ProfileMenu
  - Tabs
  - Toolbar
  - Wizard
  - NavMenu
  - NavMenuTree
  - Tree View
  - KeyCode
  - Popover

# Notification
  - MessageBar
  - MessageBox
  - Badge
    - Badge
    - CounterBadge
    - PresenceBadge
  - Toast
  - Skeleton
  - Progress
    - Progress
  - Progress Ring
  - Tooltip
  - Persona

# Data
  - Data grid
  - Paginator (not in docuemntation)
  - Flipper

# Components
  - Button
    - Button
	- MenuButton
  - SplashScreen
  - Drag and Drop
  - Pull to refresh 
  - Highlighter
  - Label
  - Overflow
  - Overlay
  - Horizontal Scroll
  
# Incubation lab
  - Overview
  - MarkdownSection
  - TableOfContents

**Comment by vnbaaij** [comment]
I think, in general this is a good restructuring. Bit more 'glanceable' sections. 

We will certainly revisit this for the next major (v5) version of the library!

**Reply by franklupo** [reply]
There are undocumented components that I would like to see: 

- CollapsibleRegion 
- Paginator 
 
The components section needs to be reviewed

**Reply by vnbaaij** [reply]
> There are undocumented components that I would like to see:
> 
> * CollapsibleRegion
> * Paginator
> 
They both need their own page and example(s) before we can add them to the menu

> The components section needs to be reviewed

How do you mean? And is that something that we need to invest time in **now** specifically. Is something broken?
